From efb3c226fc80243729901f675568b2b41142b1f9 Mon Sep 17 00:00:00 2001
From: Yong Li <yong.b.li@linux.intel.com>
Date: Mon, 18 Mar 2019 22:54:41 +0800
Subject: [PATCH] Add expiredTimerUse property support

When the timer expired, set expiredTimerUse property as the timer use

Signed-off-by: Yong Li <yong.b.li@linux.intel.com>
---
 watchdog.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/watchdog.cpp b/watchdog.cpp
index aed0443..d529746 100644
--- a/watchdog.cpp
+++ b/watchdog.cpp
@@ -101,6 +101,8 @@ void Watchdog::timeOutHandler()
         action = fallback->action;
     }
 
+    WatchdogInherits::expiredTimerUse(WatchdogInherits::currentTimerUse());
+
     auto target = actionTargetMap.find(action);
     if (target == actionTargetMap.end())
     {
-- 
2.7.4

